source/install/self-managed-changelog.md

```diff
@@ -6,12 +6,107 @@ See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release
 
 Latest Mattermost Releases:
 
+- [Release v9.1 - Feature Release](#release-v9-1-feature-release)
 - [Release v9.0 - Major Release](#release-v9-0-major-release)
 - [Release v8.1 - Extended Support Release](#release-v8-1-extended-support-release)
 - [Release v8.0 - Major Release](#release-v8-0-major-release)
-- [Release v7.10 - Feature Release](#release-v7-10-feature-release)
 - [Release v7.8 - Extended Support Release](#release-v7-8-extended-support-release)
 
+## Release v9.1 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release day: October 16, 2023**
+
+### Important Upgrade Notes
+ - Improved performance on data retention ``DeleteOrphanedRows`` queries. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for notes on a new migration that was added. Removed feature flag ``DataRetentionConcurrencyEnabled``. Data retention now runs without concurrency in order to avoid any performance degradation. Added a new configuration setting ``DataRetentionSettings.RetentionIdsBatchSize``, which allows admins to to configure how many batches of IDs will be fetched at a time when deleting orphaned reactions. The default value is 100.
+
+**IMPORTANT:** If you upgrade from a release earlier than v9.0, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Compatibility
+ - Updated Chromium minimum supported version to 116+.
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a **Cancel** button to the **Delete category** modal.
+ - Added the ability to resize the channel sidebar and right-hand sidebar.
+ - Added two new filtering options (show all channel types, show private channels) to the **Browse channels** modal.
+ - Pre-packaged GitLab plugin version v1.7.0.
+ - Added additional reaction options when viewing threads or messages when the sidebar is larger than its minimum width.
+ - Added block changes to name, display name, and purpose for direct and group messages.
```

Comment:
I was trying to look for changes in the interface, and I was unable to understand what this means and how do I access that. Probably a better description of this change is needed here to make it understandable for every body.

Reply:
The release note comes from https://github.com/mattermost/mattermost/pull/24199. @larkox @johndavidlugtu is there a better way to describe this change?

Reply:
Maybe something in the lines of:

- Improve the API to block any changes to direct and group messages name, display name or purpose. (since those are not meant to be changed)

Reply:
Ok, then I think that shouldn't be in the "UI" section, because that is not a UI related thing, it is affecting only the backend (AFAIK).

Reply:
I've moved it to the Bug Fixes section as I believe it was a bug fix.